LOCAL_DEVELOPMENT.md

```
@@ -0,0 +1,265 @@
+
+### 3a — Start the Supabase stack
+
+```bash
+supabase start
+```
+
+This command starts Postgres, Auth, Storage, and the Supabase Studio dashboard locally using Docker. On first run it may take a few minutes to pull images.
+
+Once started, the CLI prints your local credentials:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)